tuneme/templates/profiles/register.html

```diff
@@ -9,17 +9,17 @@
 <div class="block turq">
   <div class="signin-form">
     <p>{% trans "Not a member yet? Join us by signing up below." %}</p>
-    {% if messages %}
-      <ul class="messages">
-          {% for message in messages %}
-          <li>{{ message }}</li>
-          {% endfor %}
-      </ul>
-    {% endif %}
+
     <form method="post" action="{% url 'molo.profiles:user_register' %}">{% csrf_token %}
         <p><strong>{% trans "Choose a username" %}</strong></p>
+        {% if form.username.errors %}
```

Comment:
Hmn.. for consistency, lets make the rest `{{form.}}`` instead of using `{{registration_form}}`